include/mgp.hpp

```diff
@@ -4360,6 +4361,18 @@ inline List RunTextSearchQuery(mgp_graph *memgraph_graph, std::string_view index
   return results_or_error["search_results"].ValueList();
 }
 
+inline std::string AggregateOverTextIndex(mgp_graph *memgraph_graph, std::string_view index_name,
+                                          std::string_view search_query, std::string_view search_mode) {
+  auto results_or_error = Map(mgp::MemHandlerCallback(graph_aggregate_over_text_index, memgraph_graph,
+                                                      index_name.data(), search_query.data(), search_mode.data()));
+  auto maybe_error = results_or_error["error_msg"].ValueString();
+  if (!maybe_error.empty()) {
+    throw TextSearchException(maybe_error.data());
+  }
+
+  return results_or_error["aggregation_results"].ValueString().data();
```

Comment:
It returns a `std::string_view`, whereas this function returns a `std::string`. I’ve changed it so that this function also returns a `std::string_view`, and that I call `.data()` on the result only in the query module, in line with the other arguments/return values.